fakeredis/_server.py

```diff
@@ -2315,6 +2319,39 @@ def time(self):
         now_us %= 1000000
         return [str(now_s).encode(), str(now_us).encode()]
 
+    # Generic commands
+
+    @command((Key(),))
```

Comment:
I think you need the following to make this do the right thing if the key doesn't exist (which also needs a test):
```suggestion
    @command((Key(missing_return=None),))
```